internal/config/mount_config.go

```diff
@@ -24,9 +24,17 @@ type LogConfig struct {
 	FilePath string      `yaml:"file-path"`
 }
 
+type FileCacheConfig struct {
+	Path                    string `yaml:"path"`
```

Comment:
path and cachefileForRandomReads - I would suggest to use the names mentioned in design doc until we finalize the names.